apis/src/providers/games_search_context.rs

```diff
@@ -84,10 +86,11 @@ pub fn load_games(
         color: filters.color,
         result: filters.result,
     });
+    let player2 = filters.opponent.clone().map(|username| PlayerFilter { username, color: None, result: None });
```

Comment:
Currently, it only works if it is an exact match, for example: "test_user" does not match "test_user2"
What do you guys think?